aspnetcore/fundamentals/startup.md

```diff
@@ -16,19 +16,19 @@ uid: fundamentals/startup
 
 By [Steve Smith](http://ardalis.com) and [Tom Dykstra](https://github.com/tdykstra/)
 
-The `Startup` class configures the request pipeline that handles all requests made to the application.
+The `Startup` class configures services required by the app and the request pipeline that handles all requests made to the application. 
```

Comment:
Because new users may not know *what* services or what a "request pipeline" is, mostly. But I've shortened it per your feedback.